### PR TITLE
Updated Modern UI

### PR DESCRIPTION
**Pull request by Vanceagher**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -89,109 +85,25 @@ <h3><em>Convert a Scratch project to HTML</em></h3>
...
-      <ul>
-        <li>Variable and list watchers</li>
-      </ul>
-      <h3>2019-02-09</h3>
-      <ul>
-        <li>Project start</li>
-      </ul>
+      
       <p>Made by <a href="https://scratch.mit.edu/users/Sheep_maker/">Sheep_maker</a>, who used <a href="https://github.com/LLK/scratch-vm/">scratch-vm</a>, <a href="http://danml.com/download.html">download.js</a>, and their dependencies for this project.</p>
-      <p>CSS by <a href="https://scratch.mit.edu/users/mrcringekidyt/">Mr. Cringe Kid</a>.</p>
```
Comment (SheepTester) [inline index.html:194]: It seems some portion of the CSS is still theirs; I wouldn't remove their credit

File: index.html [hunk 2]
```diff
@@ -89,109 +85,25 @@ <h3><em>Convert a Scratch project to HTML</em></h3>
...
       <p id="note-1"><a href="#ref-1"><sup>1</sup></a>You may have to deal with privacy laws around cookies outside of Scratch.</p>
       <p id="note-2"><a href="#ref-2"><sup>2</sup></a>Compatibility mode forces projects to run at 30 FPS, like in Scratch 2.0. Turning this off allows the project to run at a higher framerate (usually 60 FPS, depending on the computer screen's refresh rate).</p>
       <p id="note-3"><a href="#ref-3"><sup>3</sup></a>I think the implementation of this is poor. Maybe instead of setting mouse x/y, it can set a cloud variable with a certain name. You can leave feedback and suggestions on <a href="https://scratch.mit.edu/users/Sheep_maker/">my profile</a>.</p>
       <p id="note-4"><a href="#ref-4"><sup>4</sup></a>If a cloud variable with the name "☁ eval" is set, it'll run the variable value as JavaScript and store it in a cloud variable named "☁ eval output"; if there is an error, it'll be stored in "☁ eval error." If you're using a custom cloud server, then cloud variables whose names start with "☁ local storage" will store their values in localStorage instead of in the server.</p>
-      <h2>Update history</h2>
```
Comment (SheepTester) [inline index.html:109]: Hmm, why did you remove the update history? Ideally the changelog should be accessible at least somewhere

Comment (Vanceagher) [reply index.html:109]: I honestly have no idea why.

File: index.html (hunk 1 above)
Comment (Vanceagher) [reply index.html:194]: True.

File: css.txt [hunk 3]
```diff
@@ -0,0 +1,137 @@
+@import url(https://fonts.googleapis.com/css2?family=Helvetica:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap);
+
+@font-face { 
+  font-family: 'ScratchFont';
+  src: url('scratchfont.ttf');
+}
```
[the remark this answers is not in the document]
Comment (Vanceagher) [reply css.txt:6]: They give the code to use it on a web page, I'm not sure.

File: _redirects [hunk 4]
```diff
@@ -0,0 +1 @@
+/* https://scratchtml.netlify.app/
```
[the remark this answers is not in the document]
Comment (Vanceagher) [reply _redirects:1]: I hosted a custom version on Netlify for myself, I did this a long time ago, not sure why I send this pull request.

**Comment by SheepTester** [comment]
> I hosted a custom version on Netlify for myself, I did this a long time ago, not sure why I send this pull request.

In that case, I'll just close this PR
